administration/src/cards/extensions/BirthdayExtension.tsx

```diff
@@ -45,6 +61,17 @@ const BirthdayForm = ({
         isValid={isValid || !showErrorMessage}
         maxDate={new Date()}
         disableFuture
+        sideComponent={
+          <>
+            {projectConfig.projectId.includes('koblenz') && (
```

Comment:
this is not a very secure check, i would prefer to add a flag in `projectConfig` like
`showBirthdayMinorHint` or sth similar and set it true or false